python/cuda_cccl/pyproject.toml

```diff
@@ -16,27 +16,49 @@ classifiers = [
   "License :: OSI Approved :: Apache Software License",
 ]
 requires-python = ">=3.9"
+
 dependencies = [
   "numba>=0.60.0",
   "numpy",
-  "cuda-bindings>=12.9.1,<13.0.0",
   "cuda-pathfinder>=1.1.0",
   "cuda-core",
-  "nvidia-cuda-nvrtc-cu12",
-  "nvidia-nvjitlink-cu12",
   "numba-cuda>=0.18.0",
 ]
+
 dynamic = ["version"]
 readme = { file = "README.md", content-type = "text/markdown" }
 
 [project.optional-dependencies]
-test = [
+cu12 = [
+  "cuda-bindings>=12.9.1,<13.0.0",
+  "nvidia-cuda-nvrtc-cu12",
+  "nvidia-nvjitlink-cu12",
+]
+cu13 = [
+  "cuda-bindings>=13.0.0,<14.0.0",
+  "nvidia-cuda-nvrtc",
+  "nvidia-nvjitlink",
+]
+test-cu12 = [
+  "cuda-bindings>=12.9.1,<13.0.0",
+  "nvidia-cuda-nvrtc-cu12",
+  "nvidia-nvjitlink-cu12",
```

Comment:
```suggestion
  "cuda-cccl[cu12]",
```

Reply:
Is this really legal?

Reply:
Yes, learned this from Keith and applied it to cuda-python/numba-cuda already 🙂

Reply:
That makes me even less convinced it's fully legal :P @kkraus14

Reply:
https://github.com/pypa/pip/issues/11296

This is supported but currently undocumented by pip 😁